ccd-definition/AuthorisationCaseField/CareSupervision/superuser.json

```diff
@@ -20,6 +20,13 @@
     "UserRole": "caseworker-publiclaw-superuser",
     "CRUD": "CRUD"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "hiddenCaseManagementOrders",
+    "UserRole": "caseworker-publiclaw-superuser",
+    "CRUD": "CRUD"
```

Comment:
do we ever delete from this collection of "Removed case management orders" - if not then D permission wouldn't be needed?

Reply:
We delete this collection in removeOrderController about to submit.